src/Core/Framework/Api/ApiDefinition/Generator/Schema/StoreApi/components/schemas/SalesChannelContext.json

```diff
@@ -118,6 +118,49 @@
                             }
                         }
                     },
+                    "itemRounding": {
```

Comment:
these items are always present, should be mentioned in `required`:

`"required": ["salesChannel", "apiAlias"]` -> `"required": ["salesChannel", "apiAlias", "itemRounding", "totalRounding", "languageInfo"]`

Reply:
It should be in line 169, but I cannot comment on that line ;)